xclim/utils.py

```diff
@@ -62,6 +64,275 @@
              '360_day': 360}
 
 
+def create_ensemble(ncfiles, dim='sim'):
+    """Create an xarray datset of ensemble of climate simulation from a list of netcdf files. Input data is
+    concatenated along a newly created data dimension (default=='sim')
```

Comment:
I suggest using realization: https://cf-trac.llnl.gov/trac/ticket/142

Reply:
A bit late for the party but ... il me semble que le mot realization suggère que l'ensemble est formé de différents membres d'un même modèle alors que formellement l'ensemble pourrait contenir un seul membre de différents modèles. Il me semble que "ensemble_member_label" serait mieux.